### PR TITLE
add some DMS best practices

### PR DESCRIPTION
**Pull request by taroface**
DOC-9384
DOC-9222
DOC-9062
DOC-9245

### CODE REVIEW

**Comment by taroface** [comment]
@Jeremyyang920 @amruss Friendly ping for review!

File: src/current/v22.2/aws-dms.md [hunk 1]
```diff
@@ -43,8 +43,10 @@ Complete the following items before starting this tutorial:
...
+- Manually create all schema objects in the target CockroachDB cluster, and drop all [constraints]({% link {{ page.version.version }}/constraints.md %}) per the [AWS DMS best practices](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_BestPractices.html#CHAP_BestPractices.Performance). You can recreate them after the [full load completes](#step-3-verify-the-migration).
+
+    AWS DMS can create a basic schema, but does not create [indexes]({% link {{ page.version.version }}/indexes.md %}) or constraints such as [foreign keys]({% link {{ page.version.version }}/foreign-key.md %}) and [defaults]({% link {{ page.version.version }}/default-value.md %}).
+    - If you are migrating from PostgreSQL, MySQL, Oracle, or Microsoft SQL Server, [use the **Schema Conversion Tool**](https://www.cockroachlabs.com/docs/cockroachcloud/migrations-page) to convert and export your schema. Ensure that any schema changes are also reflected on your tables, or add [transformation rules](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Tasks.CustomizingTasks.TableMapping.SelectionTransformation.Transformations.html). If you make substantial schema changes, the AWS DMS migration may fail.
```
Comment (Jeremyyang920) [inline src/current/v22.2/aws-dms.md:49]: Not sure the best way to note this but using the SCT still requires the user to drop constraints as the SCT coverts the existing schema 1:1 unless they uploaded a schema without constrains.

**Comment by taroface** [comment]
@Jeremyyang920 PTAL -- restructured the "drop constraints" bullet and also added notes on parallelization and follower reads.

**Review by Jeremyyang920 — APPROVED** [review]
Nice! Changs looks good to me.

**Review by Amruta-Ranade — APPROVED** [review]

**Pull request merged**